doc/rfcs/00000-rfc-process.md

```diff
@@ -0,0 +1,150 @@
+# RFC: RFC process
+
+The request for comments (RFC) process is intended to provide a consistent and
+controlled path for new features to enter Node.js. The process helps find
+consensus among Node.js community and collaborators on substantial changes to
+the runtime.
+
+## Motivation
+
+Attempting to contribute substantial features to Node.js can result in
+interminable, chaotic, and repetitive conversations with no clear resolution and
+no realistic process for resolution. The intention of this proposal is to
+provide a clearer process.
+
+Examples that may have benefited from an RFC process:
+
+* [Implement window.fetch into core][]
```

Comment:
I... think `window.fetch` has been blocked on the same thing all along - someone actually doing all the work to get `window.fetch` into core.

When we had the half-formal process (of talking about it in 2 collaborator summits) we were always in agreement (mostly). I am happy to dig up the notes from those meetings if that's helpful.

The thing blocking fetch in core has always been the fact that providing a reasonable implementation is a lot more work than it seems and we didn't actually want to ship a broken one that doesn't actually behave like browser fetch in many cases (like Deno's). If someone feels strongly about shipping a broken fetch and fixing things as we go along later - that is also something collaborators were OK with.

That said - we've landed thousands of lines of code in core already to enable fetch in core including things (like AbortController) that don't actually work in Deno or other runtimes (yet, I'm sure they'll fix it! At least I'm rooting for it). Users have always had a reasonable workaround (use a polyfill).

Reply:
Other ideas that could benefit from an RFC process though: 
 - Switching our API defaults (somehow, eventually) to promises.
 - Modules, the module team governance and process ended up being very frustrating to many and several collaborators quit the project.
 - The whole `unhandledRejection` thing and discussion that eventually only happened (after endless discussion) through Mary's hard work and diplomacy.
 - Promise APIs (everywhere)
 - Stuff like HTTP3

That said, I believe in most of them it's important the RFC process happens _swiftly_ and _after_ we have an initial implementation landed.